tools/jenkins/run_performance.sh

```diff
@@ -37,4 +37,13 @@ BENCHMARKS_TO_RUN="bm_closure bm_cq bm_call_create bm_error bm_chttp2_hpack bm_c
 # Enter the gRPC repo root
 cd $(dirname $0)/../..
 
-tools/profiling/microbenchmarks/bm_diff.py -d origin/$ghprbTargetBranch -b $BENCHMARKS_TO_RUN
+AFFECTS_C_CPP=`python -c 'import sys; \
```

Comment:
I'd recommend isolating this into a script somewhere... 

It's likely to get tweaked by people over time, and copy and pasted differently in different places, resulting in a maintenance nightmare.